doc/api/stream.md

```diff
@@ -1244,13 +1244,15 @@ changes:
     description: The `chunk` argument can now be a `Uint8Array` instance.
 -->
 
-* `chunk` {Buffer|Uint8Array|string|any} Chunk of data to unshift onto the
+* `chunk` {Buffer|Uint8Array|string|null|any} Chunk of data to unshift onto the
   read queue. For streams not operating in object mode, `chunk` must be a
   string, `Buffer` or `Uint8Array`. For object mode streams, `chunk` may be
-  any JavaScript value other than `null`.
+  any JavaScript value.
```

Comment:
This change isn't correct. `unshift(null)` will always end the stream no matter whether the stream is in object mode or not, so you can't use `null` as it's a reserved value.

Reply:
Did only copy from .push doc, so should we change both to include "other than null"?

Reply:
I would say so since `push()` and `unshift()` currently share implementations (minus positioning of course).